code/datums/uplink_items/uplink_traitor.dm

```diff
@@ -517,6 +517,7 @@
 	cost = 100
 	item = /obj/item/storage/box/syndie_kit/contractor
 	excludefrom = list(UPLINK_TYPE_NUCLEAR, UPLINK_TYPE_SST)
+	uses_special_spawn = TRUE
```

Comment:
Can you apply the special spawning from `UPLINK_SPECIAL_SPAWNING` here? Then LGTM